tests/playwright/src/model/pages/images-page.ts

```diff
@@ -66,10 +66,14 @@ export class ImagesPage extends MainPage {
     return await editImagePage.renameImage(newname);
   }
 
-  async startContainerWithImage(image: string, containerName: string): Promise<ContainersPage> {
+  async startContainerWithImage(
```

Comment:
This should not return void, it should return the object that it navigates to after it performs whatever function it has.